tests/Pinta.Core.Tests/BitMaskTest.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Collections.Generic;
+using Gtk;
```

Comment:
Is this unused?

Reply:
Yes, it is unused...I don't know how it ended up there. I just removed it